esmvalcore/cmor/_fixes/cmip6/fgoals_g3.py

```diff
@@ -51,7 +52,7 @@ class Mrsos(Fix):
     def fix_metadata(self, cubes):
         """Fix metadata.
 
-        FGOALS-g3 mrsos data contains error in co-ordinate bounds.
+        FGOALS-g3 mrsos data contains error in coordinate bounds.
```

Comment:
watching Romania-Ukraine as I review this, nearly jumped off me chair at this :grin: